### PR TITLE
Added method to update instance IPs

### PR DESCRIPTION
**Pull request by trenton42**
- Reverse DNS records could be edited on regular IPs, but not instances IPs

### CODE REVIEW

File: instance_ips.go [hunk 1]
```diff
@@ -104,3 +104,30 @@ func (c *Client) AddInstanceIPAddress(ctx context.Context, linodeID int, public
 
 	return r.Result().(*InstanceIP), nil
 }
+
+// UpdateInstanceIPAddress updates the IPAddress with the specified instance id and IP address
+func (c *Client) UpdateInstanceIPAddress(ctx context.Context, id int, ipAddress string, updateOpts IPAddressUpdateOptions) (*InstanceIP, error) {
```
[the remark this answers is not in the document]
Comment (trenton42) [reply instance_ips.go]: You are referring to the `id` argument, correct?

**Comment by trenton42** [comment]
Hi @displague, you are right, these two endpoints do appear to be duplicated.The thing that threw me off is this comment:

https://github.com/linode/linodego/blob/42d84d42f3f28fe1fe1823c58c11715241eda24e/network_ips.go#L65-L66

It appears that you need an id of an IP address to do the update. Looking more closely at the [documentation][0], I can see that this endpoint actually takes the IP address itself.

I'm not sure why, then, there are two endpoints that seem to do the exact same thing: [updateIP][0] and [updateLinodeIP][1]

[0]: https://developers.linode.com/api/docs/v4#operation/updateIP
[1]: https://developers.linode.com/api/docs/v4#operation/updateLinodeIP

Feel free to close this, but I think it would be helpful to change the docs on the `updateIPAddress` method.

**Comment by displague** [comment]
I totally agree @trenton42.  That `id` variable should be renamed to `address`.  I confused myself when I was comparing methods.

For a 1:1 API mapping, your pull request is still useful.  This endpoint does add the effect of only manipulating the IP if it is assigned to the specified LinodeID.  That might save someone a lookup and condition in the future.

File: instance_ips.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (trenton42) [reply instance_ips.go]: Updated!

**Comment by trenton42** [comment]
@displague Thanks again for looking over this! I made the change to the `id` parameter as requested.

**Pull request merged**